### DISCUSSION TITLE
Wolfram-mode

### DISCUSSION THREAD

**Discussion by ruanpereira** [syl20bnr/spacemacs#16027] [Q&A] [answered]
Is wolfram-mode enable in spacemacs? Still works? I saw a older release saying that wolfram-mode was removed because of melpa, but in melpa, currently, wolfram package exists. So, it works here? If so, how?

**Comment by dalanicolai** [comment, marked as answer]
You could have a look at/consider using the mathematica layer listed [here](https://github.com/syl20bnr/spacemacs/discussions/14757#discussioncomment-714850).
